backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/domain/mapper/SubscriptionMapper.java

```diff
@@ -32,4 +35,8 @@ public default List<SubscriptionEntity> toEntity(@Nullable Iterable<Subscription
 	@Mapping(target = "isNew", ignore = true)
 	SubscriptionEntity toEntity(@Nullable Subscription subscription);
 
+	@Mapping(target = "isNew", ignore = true)
+	@Mapping(target = "alertType", source = "alertType")
```

Comment:
Is this annotation required? It appears to do nothing (unless I'm mistaken).

Reply:
removed